src/CISDSC/dscresources/CIS_Microsoft_Windows_10_Enterprise_Release_20H2/CIS_Microsoft_Windows_10_Enterprise_Release_20H2.psd1

```diff
@@ -4,7 +4,7 @@
 RootModule = 'CIS_Microsoft_Windows_10_Enterprise_Release_20H2.schema.psm1'
 
 # Version number of this module.
-ModuleVersion = '1.10.1'
+ModuleVersion = '1.10.0.1'
```

Comment:
What versioning style are you using?
 
Semantic Versioning usually is x.y.z and not x.y.z.a